ui/redux/actions/comments.js

```diff
@@ -39,6 +41,102 @@ export function doCommentList(uri: string, page: number = 1, pageSize: number =
   };
 }
 
+export function doCommentReactList(uri: string | null, commentId?: string) {
+  return (dispatch: Dispatch, getState: GetState) => {
+    const state = getState();
+    const channel = localStorage.getItem('comment-channel');
+    // if not channel, fail?
+    if (!channel) {
+      dispatch({
+        type: ACTIONS.COMMENT_REACTION_LIST_FAILED,
+        data: 'No active channel found',
+      });
+      return;
+    }
+    const commentIds = uri ? makeSelectCommentIdsForUri(uri)(state) : [commentId];
+    const myChannels = selectMyChannelClaims(state);
+    const claimForChannelName = myChannels.find(chan => chan.name === channel);
+    const channelId = claimForChannelName && claimForChannelName.claim_id;
+    dispatch({
+      type: ACTIONS.COMMENT_REACTION_LIST_STARTED,
+    });
+    Lbry.comment_react_list({
+      comment_ids: commentIds.join(','),
+      channel_name: channel,
+      channel_id: channelId,
+    })
+      .then((result: CommentReactListResponse) => {
+        const { my_reactions: myReactions, others_reactions: othersReactions } = result;
+        dispatch({
+          type: ACTIONS.COMMENT_REACTION_LIST_COMPLETED,
+          data: {
+            myReactions,
+            othersReactions,
+          },
+        });
+      })
+      .catch(error => {
+        dispatch({
+          type: ACTIONS.COMMENT_REACTION_LIST_FAILED,
+          data: error,
+        });
+      });
+  };
+}
+
+export function doCommentReact(commentId: string, type: string) {
+  return (dispatch: Dispatch, getState: GetState) => {
+    const state = getState();
+    const channel = localStorage.getItem('comment-channel');
+    // if not channel, fail?
+    if (!channel) {
+      dispatch({
+        type: ACTIONS.COMMENT_REACTION_LIST_FAILED,
+        data: 'No active channel found',
+      });
+      return;
+    }
+    const myChannels = selectMyChannelClaims(state);
+    const myReacts = makeSelectMyReactionsForComment(commentId)(state);
+    const claimForChannelName = myChannels.find(chan => chan.name === channel);
+    const channelId = claimForChannelName && claimForChannelName.claim_id;
+    const exclusiveTypes = {
+      [REACTION_TYPES.LIKE]: REACTION_TYPES.DISLIKE,
+      [REACTION_TYPES.DISLIKE]: REACTION_TYPES.LIKE,
+    };
+
+    dispatch({
+      type: ACTIONS.COMMENT_REACT_STARTED,
+    });
+    const params: CommentReactParams = {
+      comment_ids: commentId,
+      channel_name: channel,
+      channel_id: channelId,
+      react_type: type,
+    };
+    if (Object.keys(exclusiveTypes).includes(type)) {
+      params['clear_types'] = exclusiveTypes[type];
+    }
+    if (myReacts.includes(type)) {
+      params['remove'] = true;
+    }
+
+    Lbry.comment_react(params)
+      .then((result: CommentReactListResponse) => {
+        dispatch({
+          type: ACTIONS.COMMENT_REACT_COMPLETED,
+        });
+        dispatch(doCommentReactList(null, commentId));
```

Comment:
We should just populate the new reaction count here, instead of calling another API
https://github.com/lbryio/lbry-desktop/blob/odysee/ui/redux/reducers/reactions.js#L29-L53